marketpulse/main/fxosprice_urls.py

```diff
@@ -4,6 +4,6 @@
 urlpatterns = patterns(
     '{0}.main.views'.format(settings.PROJECT_MODULE),
     url(r'^new/$', 'edit_fxosprice', name='edit_fxosprice'),
-    url(r'^(?P<username>[A-Za-z0-9_]+)/(?P<id>\d+)/$',
-        'fxosprice_home', name='fxosprice_home'),
+    url(r'^view/$', 'view_fxosprice', name='view_fxosprice'),
```

Comment:
I would prefer to use something similar to ``activity/`` which seems to be more descriptive about the page.